.github/workflows/goreleaser.yml

```diff
@@ -75,9 +75,10 @@ jobs:
 
       - name: Run GoReleaser
         id: run-goreleaser
-        uses: goreleaser/goreleaser-action@8f67e590f2d095516493f017008adc464e63adb1 # v4.1.0
+        uses: goreleaser/goreleaser-action@v6 # v4.1.0
         with:
-          version: v1.26.2
+          distribution: goreleaser
+          version: 'v2'
```

Comment:
Should we ping to a specific version instead of the major version `v2` to avoid surprises with automatic version changes? Open to discussion / thoughts here.